src/alire/alire-config.ads

```diff
@@ -25,6 +25,10 @@ package Alire.Config with Preelaborate is
 
       Editor_Cmd  : constant Config_Key := "editor.cmd";
 
+      Catalog_Autoconfig : constant Config_Key := "catalog.autoconfig";
```

Comment:
I don't find the name very meaningful. I would rather have something like `catalog.auto_community`.
Also do we use the word `catalog` a lot? I thought `index` was the right word here.

Reply:
I was trying to go for a catalog being the sum of all your configured indexes, but it's probably too confusing, yeah.